skimage/feature/texture.py

```diff
@@ -48,6 +50,8 @@ def graycomatrix(image, distances, angles, levels=None, symmetric=False, normed=
         by the total number of accumulated co-occurrences for the given
         offset. The elements of the resulting matrix sum to 1. The
         default is False.
+    mask : array_like, optional
```

Comment:
If you make `mask` an `array_like` it should be later wrapped in `np.asarray(, dtype=bool)`.

The current implementation actually requires a full boolean numpy array. So it should read
```suggestion
    mask : bool array, optional
```

I'd prefer the later because it's more explicit and will lead to fewer edge cases.